docs/dex-blog-post.md

```diff
@@ -157,29 +83,93 @@ The seller contract guarantees that the seller box can be spent:
 2) by a _swap transaction_ created by Matcher in which _seller box_ is spent together (i.e.
 atomically) with the matched _buyer box_ (see [buy tokens](#buy-tokens)).
 
```

Comment:
@aslesarenko It's up to date. Check out the "live" version - https://github.com/ergoplatform/ergo-contracts/blob/992ad3e27e025761dd8708fceb5f54940b7d85a0/verified-contracts/src/main/scala/org/ergoplatform/contracts/AssetsAtomicExchange.scala#L12-L47
It just has checks `(OUTPUTS.size > 1 && OUTPUTS(1).R4[Coll[Byte]].isDefined)` removed. They were added to make Stainless happy. I've also skipped them in the buyer contract in the post for consistency. Do you want to put them back?

Reply:
I added `ergAmount` parameter description.

Reply:
It think the contracts in the post should be exactly the same as they are used in the implementation. 1) to avoid confusion 2) to serve as working examples, which people can reproduce

Reply:
Agreed. Copied contracts from the repo and added links.